docs/statcast_fielding.md

```diff
@@ -0,0 +1,123 @@
+# Statcast Fielding Outs Above Average
+`statcast_outs_above_average(year: int, pos: str, min_att: Union[int, str] = "q")`
+
+This function retrieves outs above average (OAA) for the given year, position, and attempts. OAA is a Statcast metric based on the "cumulative effect of all individual plays a fielder has been credited or debited with, making it a range-based metric of fielding skill that accounts for the number of plays made and the difficulty of them".
+
+## Arguments
+`year:` The year for which you wish to retrieve batted ball against data. Format: YYYY.
+`pos:` The position you are interested in. Valid positions include "all", "IF", "OF", and position names or abbreviations. Pitchers and catchers are not included.
```

Comment:
Somewhat unrelated parenthetical: who the hell would rather use the position names???

Reply:
Not sure, but I included them in case anyone does. I think motivated by how we handled the pitch names. Could maybe switch it up to accept position numbers?

Reply:
No need to remove, maybe adding numbers could be good 🤷